Main.cs

```diff
@@ -682,8 +705,23 @@ public static void Run(string[] _args)
                     wsvc.OnStart(args.ToArray());
                     Thread.Sleep(1000);
                     wsvc.OnStop();
+                    return;
+                } 
+                if (args[0] == "help")

```

Comment:
This should probably support \? and -? as well.  Common syntax across all console applications